app/index.jsx

```diff
@@ -63,6 +63,17 @@ const requestSubscribe = streamManager.subscribe.bind(streamManager);
 const requestForget = streamManager.forget.bind(streamManager);
 const shareOrigin = window.location.href.split('?')[0];
 
+
+// Update menu height to show chartControl on top of dialog in mobile devices.
+const style = '.smartcharts-mobile .cq-chart-controls .ciq-menu .cq-menu-overlay';
```

Comment:
do you need to do this? Would just modifying `app.scss` work?

Reply:
How else i can use javascript in css? I need `window.innerHeight` to calculate the actual vh in ios devices.its why i said we should use props or css variables, otherwise i have to overwrite the style using js.

Reply:
true that